docs/install.rst

```diff
@@ -1,103 +1,134 @@
-.. _install:
-
-************
+============
 Installation
-************
+============
+
+This page has instructions on installing OPS, installing software packages
+that integrate with OPS, and testing that your OPS installation is working.
 
-OpenPathSampling currently only works on Mac and Linux. It is tested against
-Python 2.7, 3.6, and 3.7, although there may be some corners of the code
-that aren't Python 3-compatible yet.
+OpenPathSampling currently only works on POSIX systems (macOS and
+UNIX/Linux). It is tested against Python 2.7, 3.7, 3.8, and 3.9.
 
-.. note:: As of OpenPathSampling 1.1, OpenMM will no longer be automatically
-          installed when you install OPS. However, the OpenMM engine will be
-          immediately available if you install OpenMM yourself. See the
-          `OpenMM installation instructions
-          <http://docs.openmm.org/latest/userguide/application.html#installing-openmm>`_
-          for a detailed guide, but ``conda install -c conda-forge -c omnia
-          openmm`` will work for most people. (COMING SOON: details on OPS
-          integrations with other tools.)
+Installing OpenPathSampling
+===========================
 
 .. _install-with-conda:
 
-Standard Install with Conda
-===========================
+Installation with ``conda`` (recommended)
+-----------------------------------------
 
-We recommend using ``conda`` to install OpenPathSampling.  `conda
-<http://www.continuum.io/downloads>`_ is a Python package manager built for
-scientific Python, and which handles binary dependencies seamlessly.  If you
-don't want the full ``conda`` installation, the ``miniconda`` package
-provides much of the convenience of ``conda`` with a smaller footprint.
+OpenPathSampling can be installed with ``conda``, ``pip``, or by using
+``setuptools``. We recommend using ``conda`` because it provides easy
+installation of other tools, including molecular dynamics engines such as
+OpenMM and GROMACS, which most users will want to install when using OPS.
+Conda can be installed either with the `full Anaconda distribution
+<https://www.anaconda.com/products/individual>`_, or with
+the `smaller-footprint miniconda
+<https://docs.conda.io/en/latest/miniconda.html>`_. 
 
-OpenPathSampling is part of the ``conda-forge`` channel in ``conda``.  To
-install the most recent release of OpenPathSampling with conda, use the
-following command ::
+OpenPathSampling can be installed from the ``conda-forge`` channel. To
+install the most recent release, use the command ::
 
   $ conda install -c conda-forge openpathsampling
 
 With that, you should be ready to use OPS!
 
-.. _developer-install-conda:
+Installation with ``pip`` or ``setuptools``
+-------------------------------------------
+
+OPS can also be installed with ``pip`` via the command ``pip install
+openpathsampling``, or the source can be downloaded and installed using
```

Comment:
future proofing this to the recommended method (`pip install openpathsampling` in python 3.10 installs it in a python3.1 subdirectory...)
```suggestion
OPS can also be installed with ``pip`` via the command ``python -m pip install
openpathsampling``, or the source can be downloaded and installed using
```

Reply:
Taking the suggestions, but do you have a link for the dangers of straight `pip install`? I knew that `python -m` was preferred (and should have known to write that here). But I always thought it was a "make sure that `pip` is associated with the `python` you want." I hadn't heard of Python 3.10 issues.

Personally, I switched to to `python -m pip` because sometimes conda doesn't seem to `hash -r` correctly. I will often `which python` to check that it is correct in a new env, but don't think to `which pip`, and after completely borking my conda setup when installing code for a software paper I was reviewing..... Never again!

Reply:
I normally refer to [this blog post](https://snarky.ca/why-you-should-use-python-m-pip/). It is mainly making sure you are installing into the expected python (which I have seen very experienced users F up on HPC systems due to `module load` shenanigans) on top of that there is the benefit:

> And if you're on Windows there is an added benefit to using python -m pip as it lets pip update itself. Basically because pip.exe is considered running when you do pip install --upgrade pip, Windows won't let you overwrite pip.exe. But if you do python -m pip install --upgrade pip you avoid that issue as it's python.exe that's running, not pip.exe.

the 3.10 -> 3.1 issue is just something I noticed when trying to build mdtraj on 3.10 (which is possible :smile: ). I think that one got fixed recently in both pip and conda (conda also used to complain about not finding any packages for python 3.1 if you try to conda update from a python 3.10 env :laughing: )